drivers/sensor/scd30/Kconfig

```diff
@@ -0,0 +1,30 @@
+
+menuconfig SCD30
+    depends on I2C
+    select SENSIRION_COMMON
+    bool "Sensirion SCD30 CO2 Sensor"
```

Comment:
Please move this line up.

Makes it easier to see the type and prompt text of this setting.